src/Manager.js

```diff
@@ -0,0 +1,84 @@
+
+class Manager {
+  constructor(userData, roomData, bookingsData) {
+    this.users = userData;
+    this.rooms = roomData;
+    this.bookings = bookingsData;
+    this.selectedGuest;
+    // this.selectedGuestData = consolidateGuestData();
+    this.guestBookings; // GOES ON BOTH
+    // this.guestTotalSpent; // GOES ON GUEST CLASS!
+  }
+
+  selectGuest(name) {
+    this.selectedGuest = this.users.find(user => {
+      return user.name === name;
+    })
+  }
+
+  getSelectedGuestBookings() {
+    this.bookings = this.bookings.filter(booking => {
+      return booking.userID === this.selectedGuest.id;
+    })
+  }
+/*
+FOR GUEST CLASS!
+  getGuestTotalSpent() {
+    return this.rooms.reduce((totalSpent, room) => {
+      this.bookings.forEach(booking => {
+        if (booking.roomNumber === room.number) {
+          totalSpent += room.costPerNight
+      }
+    })
+      return totalSpent
+    }, 0) 
+  }
+  */
+
+  /*
+  PERHAPS USE THIS ON A PARENT CLASS??
+  consolidateGuestData() {
+    this.selectedGuestData = {
+      // id: this.selectguest.id,
+      // name: this.selectguest.name,
+      bookings: this.getSelectedGuestBookings(),
+      totalSpent: this.getGuestTotalSpent(),
+    }  
+  }
+  */
+
+  bookRoomForGuest(userID, date, roomNumber) { // possibly put on a parent class
+    let bookingData =  { userID: userID, date: date, roomNumber: roomNumber }
+    // call fetchPOST here, I think
+    return bookingData;
+  }
+
+  deleteBookingForGuest(bookingID) {
+    let bookingToCancel = {id: bookingID}
+    // call fetchDELETE here, I think
```

Comment:
I believe you're right about fetch delete here

Reply:
Thanks Aaron!